transbank/onepay/__init__.py

```diff
@@ -28,10 +28,27 @@ class IntegrationType(Enum):
     TEST = Integration("TEST","https://onepay.ionix.cl","8e279b4e-917d-4cbf-b0e3-9432adefff6a")
     MOCK = Integration("MOCK","https://transbank-onepay-ewallet-mock.herokuapp.com","04533c31-fe7e-43ed-bbc4-1c8ab1538afp")
 
+api_key = "dKVhq1WGt_XapIYirTXNyUKoWTDFfxaEV63-O5jcsdw"
+shared_secret = "?XW#WOLG##FBAGEAYSNQ5APD#JF@$AYZ"
+integration_type = IntegrationType.TEST
+callback_url = "http://no.callback.has/been.set"
+app_scheme = None
+qr_width_height = None
+commerce_logo_url = None
+
+defaultOptionsHash = {
```

Comment:
In python we call "dict" (short for 'dictionary') to what ruby folks call "hash".

So I'd rename this to `defaultOptionsDict`. Or even better, `defaultOptions`.